jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1276,6 +1276,220 @@ ecma_builtin_typedarray_prototype_fill (ecma_value_t this_arg, /**< this argumen
   return ecma_copy_value (this_arg);
 } /* ecma_builtin_typedarray_prototype_fill */
 
+/**
+ * SortCompare abstract method
+ *
+ * See also:
+ *          ECMA-262 v5, 15.4.4.11
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_typedarray_prototype_sort_compare_helper (ecma_value_t j, /**< left value */
+                                                       ecma_value_t k, /**< right value */
+                                                       ecma_value_t comparefn) /**< compare function */
+{
+  ecma_value_t ret_value = ECMA_VALUE_EMPTY;
+  ecma_number_t result = ECMA_NUMBER_ZERO;
+  bool j_is_undef = ecma_is_value_undefined (j);
+  bool k_is_undef = ecma_is_value_undefined (k);
```

Comment:
How can this be undefined for a typed array?

Reply:
I don't know either... I thought I had left this in for a reason but I guess I just forgot to remove it.